pkg/metrics/metrics.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate protoc --go_out=. --go_opt=paths=source_relative clientanalytics.proto
+//go:generate protoc --go_out=. --go_opt=paths=source_relative internal_user_log.proto
+package metrics
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/uuid"
+)
+
+const (
+	ClientTypeValue    = 16
+	LogSourceValue     = 971
+	LogSourceNameValue = "CUTTLEFISH_METRICS"
+	ToolName           = "cvdr"
+)
+
+var (
+	UserTypeVal    = UserType_GOOGLE
```

Comment:
cvdr is not only going to be used inside Google, would this user type be correct outside too?

Reply:
Thanks Jorge. 
I created this b/309514404 to follow the necessary changes for this part.

I noticed that aetst team are using git to differentiate between internal and external users , but we may need better way than this , all ideas and details wil be updated on b/309514404 later on